src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -93,6 +93,14 @@ def create(self, size, isolate_nspace, pool, mode, uid, gid):
 
         subvol_path = os.path.join(self.base_path, str(uuid.uuid4()).encode('utf-8'))
         try:
+            # create group directory with 0755
+            try:
+                self.fs.stat(self.group.path)
+            except cephfs.Error as e:
+                if e.args[0] == errno.ENOENT:
+                    self.fs.mkdirs(self.group.path, 0o755)
+                else:
+                    raise VolumeException(-e.args[0], e.args[1])
             # create directory and set attributes
             self.fs.mkdirs(subvol_path, mode)
```

Comment:
How about mkdir here?

Reply:
`mkdirs` is required here as it creates `<subvol>/<uuid>` directories.